SECURITY.md

```diff
@@ -14,6 +14,7 @@ Centreon reserves the right to make final decisions regarding publishing acknowl
 
 ## 2022
 
+* 2022/10/05 - Héctor Morales (hmorales@centreon.com)
```

Comment:
Hello denli,
If you need to insert an email address, could you do so using markdown ?

Reply:
@denliA ?

Reply:
@denliA ?